SA/design_concept/content/results/results_ui_demonstration.typ

```diff
@@ -1,27 +1,36 @@
 #import "../../../acronyms.typ": *
 
 = UI Demonstration
-@visual-fp-ui-demo-part-one and @visual-fp-ui-demo-part-two depict a step-by-step demonstration of the #ac("PoC") application
-using the `mapAdd5` code scenario as descriped in @design_eval_code_scenarios.
+
+@visual-fp-ui-demo-step-one, @visual-fp-ui-demo-step-two, @visual-fp-ui-demo-step-three, @visual-fp-ui-demo-step-four, @visual-fp-ui-demo-step-five and @visual-fp-ui-demo-step-six depict a step-by-step demonstration of the #ac("PoC") application
```

Comment:
I think something like `Figure X through Figure Y depcit ...` would be easier to read.

Reply:
Updated